src/components/Form.js

```diff
@@ -1,200 +1,184 @@
-					type: "Input",
-					isRequired: true,
-				},
-				{
-				placeholder: "Last Name",
-					camelCaseName: "lastName",
-					type: "Input",
-					isRequired: true,
-				},
-				{
-				placeholder: "Birthday",
-					camelCaseName: "birthday",
-					type: "DatePicker"
-				},
-				{
-				placeholder: "Coolness Level",
-					camelCaseName: "coolnessLevel",
-					type: "PickerInput",
-					isRequired: true,
-					options: {
-						data: ["Super Cool", "Cool", "Uncool", "Super Uncool"]
-					}
-				},
-				{
-				placeholder: "Major",
-					camelCaseName: "major",
-					type: "FilterPicker",
-					isRequired: true,
-					options: {
-						data: ["Mechanical Engineering", "Electrical Engineering", "Computer Science"],
-						type: "single"
-					}
-				}
-			]
-
-			initialValue = [
-				{
-					camelCaseName: "firstName",
-					value: "Steven"
-				},
-				{
-					camelCaseName: "lastName",
-					value: "Perdomo"
-				},
-				{
-					camelCaseName: "birthday",
-					value: "2001-01-01"
-				},
-				{
-					camelCaseName: "coolnessLevel",
-					value: "Super Uncool"
-				},
-				{
-					camelCaseName: "major",
-					value: "Computer Science"
-				}
-			]
-
-			title = ""
-
-			onSubmit = (value) => {
-				console.log(`this is the value: ${value}`)
-			}
-
-			onCancel = () => {
-				console.log("I've been cancelled")
-			}
-
-			visibile = this.state.formVisibility
-
-			changeVisibility = (visible) => {
-				this.setState({ formVisibility: visible })
-			}
-
-			submitButtonName = "Confirm Coolness Level"
-
-		Output:
-			this.state = {
-				firstName: "Haniel",
-				lastName: "Diaz",
-				birthday: null || "6969/04/20",
-				coolnessLevel:  "Super Cool",
-				major: "Computer Science"
-			}
-			onSubmit(this.state)
-
-*/
-const dimension = Dimensions.get("screen");
+import { ScrollView, SafeAreaView, Modal } from "react-native";
+import { Button, ButtonLayout, DatePicker, NavBar, Input, PickerInput, TimePicker, FilterList } from "./";
+
+/**
+ * Types
+ * @typedef {Object} Elements:
+ * 		@property {String} name
+ * 		@property {String} camelCaseName
+ * 		@property {String} type
+ * 		@property {boolean=} isRequired
+ * 		@property {any=} options
+ * @typedef InitialValue:
+ * 		@property {String} name
+ * 		@property {any} value
+ */
+
+/**
+ * Form Component Info
+ * ________________________________________________________________
+ * 	Props:
+ *		@param {Elements} elements		An array of the names of Elements.
+ *		@param {InitialValue[]=} initialValues		An array of initial Values.
+ *		@param {String} title		Displayed at the top of the form.
+ *		@param {Boolean} visible		Used to determine whether the form is visible.
+ *		@param {Function} changeVisibility		Used to change the visibility of the form.
+ *		@param {Function} onSubmit		Called to pass all the form values into.
+ *		@param {Function=} onCancel		Called when the cancel button is pressed.
+ *		@param {String=} submitButtonName		Displayed on the submit button
```

Comment:
Use spaces for indentation after initial text, not tabs, and align the description of props